ENVIRONMENT.rst

```diff
@@ -42,7 +42,7 @@ Environment Configuration Settings
 - **AWS_ENDPOINT**: (optional) in format 'https://s3.AWS_REGION.amazonaws.com:443', if not specified will be calculated from AWS_REGION
 - **WALE_S3_ENDPOINT**: (optional) in format 'https+path://s3.AWS_REGION.amazonaws.com:443', if not specified will be calculated from AWS_ENDPOINT or AWS_REGION
 - **WALE_S3_PREFIX**: (optional) the full path to the backup location on S3 in the format s3://bucket-name/very/long/path. If not specified Spilo will generate it from WAL_S3_BUCKET.
-- **WALG_DELTA_MAX_STEPS**, **WALG_DELTA_ORIGIN**, **WALG_DOWNLOAD_CONCURRENCY**, **WALG_UPLOAD_CONCURRENCY**, **WALG_UPLOAD_DISK_CONCURRENCY**: (optional) configuration options for wal-g. If at least of the options is specified, wal-g will be used instead of wal-e.
+specified, wal-g will be used instead of wal-e.
```

Comment:
wal-g is used by default only for restore

Reply:
Sure, and I'm fine with your changes since I haven written them. Just moved wal-g options into a dedicated section. Or did I miss the point?

Reply:
I think there is something wrong with formatting.
The `If at least of the options is specified, wal-g will be used instead of wal-e` sentence was split into two and one part was moved to the paragraph about wal-g.

Probably we should remove `specified, wal-g will be used instead of wal-e.` from here completely

Reply:
Got it. Changed in the latest commit.